go.mod

```diff
@@ -1,17 +1,17 @@
+// https://blog.golang.org/using-go-modules
 module github.com/freeipa/freeipa-operator
 
-go 1.13
+go 1.14
 
 require (
-	github.com/go-logr/logr v0.1.0
-	github.com/onsi/ginkgo v1.12.1
-	github.com/onsi/gomega v1.10.1
-	github.com/pkg/errors v0.9.1 // indirect
-	golang.org/x/net v0.0.0-20200904194848-62affa334b73 // indirect
-	golang.org/x/sys v0.0.0-20200909081042-eff7692f9009 // indirect
-	golang.org/x/xerrors v0.0.0-20200804184101-5ec99f83aff1 // indirect
-	k8s.io/api v0.18.6
-	k8s.io/apimachinery v0.18.6
-	k8s.io/client-go v0.18.6
-	sigs.k8s.io/controller-runtime v0.6.2
+	github.com/go-logr/logr v0.4.0
+	github.com/google/uuid v1.2.0
+	github.com/onsi/ginkgo v1.15.1
+	github.com/onsi/gomega v1.11.0
+	github.com/openlyinc/pointy v1.1.2
```

Comment:
INFO Make easier initializing k8s objects for some parameters which expect a pointer.